sale_import_base/views/sale_import_payload.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_sale_import_payload_form" model="ir.ui.view">
+        <field name="name">sale.import.payload.form</field>
+        <field name="model">sale.import.payload</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
+                        name="button_retry"
+                        type="object"
+                        string="Retry job"
+                        attrs="{'invisible': [('state', '=', 'done')]}"
+                        class="btn-primary"
+                    />
+                    <field name="state" widget="statusbar" />
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="create_date" />
+                            <field name="sale_channel_id" />
```

Comment:
@florian-dacosta FYI another thing I found useful in practice is to add a `sale_order_id` field on the sale.import.payload to be filled with the created or existing SO, and a "N Payloads" head button on the sale.channel sending to the tree view of the related payloads.